boards/v2/nuvoton/numaker_pfm/Kconfig.numaker_pfm

```diff
@@ -5,5 +5,10 @@
 # Copyright (c) 2023 Nuvoton Technology Corporation.
 
 config BOARD_NUMAKER_PFM_M467
-	bool "NUVOTON PFM M467 Development Board"
-	depends on SOC_M467
+	bool
+	select SOC_M467
+	select SOC_FLASH_NUMAKER
```

Comment:
I think this line belongs in `Kconfig` file instead.

Reply:
fixed